### PR TITLE
Use proper type for inf buffers

### PR DESCRIPTION
**Pull request by aafemt**
Also one forgotten rename.

### CODE REVIEW

File: src/include/firebird/impl/inf_pub.h [hunk 1]
```diff
@@ -184,7 +184,7 @@ enum info_features // response to fb_info_features
 	fb_feature_statement_timeout	= 6,	// Statement timeout is supported
 	fb_feature_statement_long_life  = 7,    // Prepared statements are not dropped on transaction end
 
-	info_provider_features_max      // Not really a feature. Keep this last.
+	info_features_max      // Not really a feature. Keep this last.
```
Comment (asfernandes) [inline src/include/firebird/impl/inf_pub.h:187]: I think all items of enums should have the same prefix.

Comment (aafemt) [reply src/include/firebird/impl/inf_pub.h:187]: Yes but this one is not quite item of enum and used to be used in code like "info_features array[info_features_max]".

Comment (asfernandes) [reply src/include/firebird/impl/inf_pub.h:187]: It is still an item of the enum.

Comment (AlexPeshkoff) [reply src/include/firebird/impl/inf_pub.h:187]: As a compromise suggest fb_features_max

**Comment by aafemt** [comment]
AppVeyor again failed to upgrade environment. WTH..?

**Review by AlexPeshkoff — APPROVED** [review]

**Pull request merged**